site/en/docs/webstore/using_webstore_api/index.md

```diff
@@ -2,36 +2,53 @@
 layout: "layouts/doc-post.njk"
 title: "Using the Chrome Web Store Publish API"
 #date: TODO
-#updated: TODO
+updated: 2021-12-21
 description: How to programmatically create, update, and publish items in the Chrome Web Store.
 ---
 
 ## Overview {: #overview }
 
 The Chrome Web Store Publish API provides a set of REST endpoints for programmatically creating,
-updating, and publishing items in the Chrome Web Store. Using this API, you can automate the process
-of uploading and publishing items into the store.
+updating, and publishing items in the Chrome Web Store.
 
 ## Before you begin {: #beforeyoubegin }
```

Comment:
```suggestion
## Initial setup {: #setup }
```

The title "Before you begin" is a little weird because it's a decent amount of work that you can't skip. I think "Initial setup", "Preparing the way", "API configuration", or something else might be a better framing for what we cover in this section.